scopes/envs/envs/envs.cmd.tsx

```diff
@@ -10,7 +10,7 @@ import { EnvOverview } from './components/env-overview';
 
 export class ListEnvsCmd implements Command {
   name = 'list';
-  description = 'list all envs available in the workspace';
+  description = 'lists all envs available in the workspace';
```

Comment:
@giteden that's not correct.
it doesn't show available envs, but the list of components and their envs.

Reply:
@GiladShoham you're talking about `bit envs`. `bit envs list` lists all the available envs.